README.md

```diff
@@ -1,61 +1,151 @@
 
-# 2. Configuration
 
-## 2.1 Splunk Setup Page
-An access token is required for the API access. If you are already a customer and do not have one, please do not hesitate to contact us. If you are not a customer yet, please feel free to contact us for a demo account.
+## Installation
 
-Contact Mail: ti-support [a] dcso.de
+You can install the DCSO TIE AddOn within the Splunk Enterprise Web interface:
 
-The token has to be configured in the setup page of the technical add-on on the Splunk HF. You also have to enable the script by `unchecking` the "tie2index.py" box. There are also options for the schedule and the Index where the IoC's are stored. The Index must be known on the HF.
+1. click on the `splunk>enterprise`-logo
+2. click on the wheel next next to 'Apps'
+3. click 'Install app from file'
+4. choose the file, navigating to the folder on your local machine containing a file called like `DCSO_TIE_Splunk_AddOn2-2.0.0b5.zip`
+5. if you are upgrading, make sure to check 'Upgrade app'
+6. click 'Upload'
 
-## 2.2 Standard Filter
+You can also install the add-on through the Splunk CLI (Command Line Interface):
 
-The default settings for the filter you find in default/dcso_tie_setup.conf
+```
+${SPLUNK_HOME}/bin/splunk install app DCSO_TIE_Splunk_AddOn2-2.0.0b5.zip
+```
 
+# Configuration
 
-# 3. Usage
+After installation, the add-on needs to be configured.
 
-## 3.1 Getting the IoCs
+## Splunk App Setup Page
 
-### 3.1.1 tie2index
+After installation you must setup the app or add-on.
 
-The input script tie2index.py will automatically start with the oldest IoC in a 30 day range. From that it will iterate and index all updates made. The intervall is by default 10 minutes. All IoC and their update will be stored in an index (default: dcso_app_tie-api). We recommend at least 180 days as retention time for this index. From this index all lookups and files can be derived.
+An API or Machine Token is required to access the Threat Intelligence Engine or TIE. Both the legacy
+token created through `tie.dcso.de` and the newer tokens created through `portal.dcso.de` are supported.
+If you have any questions about this Token, please contact DCSO (see below).
 
-To limit the used licence volume we only index IoCs within specified confidence and severity ranges. The ranges in the filter mentioned above are default.
+There are few more details about the configuration:
 
+* **API Token**: either a legacy tie.dcso.de token, or new one created through the DCSO Portal.
+* **Initial IoC Sequence Number**: Use this to start from a particular sequence number. This is
+  useful when re-installing or upgrading to incompatible add-on version (data in index would
+  stay compatible). Leave 0 to use whatever number is stored or start from NOW minus 30 days. 
+* **tie2index.py** script: make sure to enable this by un-checking the checkbox.
+* **Index for IoCs**: the index used to stre the IoC (events). When using a custom index, it
```

Comment:
Typo. `stre`

Reply:
Had to read twice what I meant there :) Change to `to store IoCs`.